### DISCUSSION TITLE
Does fastapi have a maximum defined http interface limit?

### DISCUSSION THREAD

**Discussion by yangpengcheng** [fastapi/fastapi#6757] [Questions] [answered]
I have defined 23 interfaces in fastapi，When I want to add the http interface to fastapi again, it is not displayed in http://hostname:port/docs, and it is returned 
{
    "detail": "Not Found"
}
when requested
I didn’t find the relevant description in the document, I was full of ...

**Comment by yangpengcheng** [comment, marked as answer]
Reopen the project, the problem is solved, I don’t know why

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue 👍
